source/sdk/io/native/FileUnix.ooc

```diff
@@ -316,5 +320,17 @@ version (unix || apple) {
 		mkfifo: override func ~withMode (mode: Int32) -> Int {
 			_mkfifo(path as CString, mode as ModeT)
 		}
+
+		createTempFile: static func (pattern, mode: String) -> FStream {
+			result: FStream
+			mask := umask(S_IRUSR | S_IWUSR)
+			fd := mkstemp(pattern)
```

Comment:
It is fixed now anyway. Maybe someday a client will come and say "I have a unix system from 1991, write me a program" :P